pkg/preparer/orchestrate.go

```diff
@@ -18,10 +18,19 @@ type Pod interface {
 	Halt() (bool, error)
 }
 
+type RealityStore interface {
+	Pod(string, string) (*pods.PodManifest, error)
+	SetPod(string, pods.PodManifest) (time.Duration, error)
+}
+type IntentStore interface {
+	RegisterPodService(pods.PodManifest) error
+	WatchPods(string, <-chan struct{}, chan<- error, chan<- pods.PodManifest) error
+}
+
 type Preparer struct {
 	node   string
-	iStore *intent.Store
-	rStore *reality.Store
+	IStore IntentStore
```

Comment:
You don't need to make these public to make them accessible in `orchestrate_test` - was that your goal?